ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -284,6 +284,9 @@ private function formatColumn(array $column, array $data, array $settings) {
     }
     if (!empty($column['show_linebreaks'])) {
       $cssClass[] = 'crm-search-field-show-linebreaks';
+      if ($column['type'] === 'html') {
+        $out['val'] = nl2br($out['val']);
+      }
```

Comment:
Seems like `nl2br` is visually doing the same thing as the `show-linebreaks` css class. So can we remove the class then?

Reply:
`show-linebreaks` is still necessary for plaintext IIRC.

Reply:
@MegaphoneJon so maybe it should be if/else?